BackofficeBundle/Tests/Functional/Controller/EditNodeControllerTest.php

```diff
@@ -38,8 +38,9 @@ public function setUp()
      *
      * @dataProvider provideMetaAndNodeId
      */
-    public function testEditRootNode($expectedMeta, $newMeta, $nodeId)
+    public function testEditNode($expectedMeta, $newMeta, $nodeId)
     {
+        $this->markTestSkipped('The tested service has been desactivated');
```

Comment:
Why did you skip that test ? Is it a temporary skip until all bundles have been merged ?